ui-admin/src/forms/designer/split/PanelEditor.tsx

```diff
@@ -0,0 +1,50 @@
+import { FormElement, FormPanel, PortalEnvironmentLanguage, Question } from '@juniper/ui-core'
+import { QuestionDesigner } from '../QuestionDesigner'
+import React, { useId } from 'react'
+import { CollapsibleSectionButton } from 'portal/siteContent/designer/components/CollapsibleSectionButton'
+import { TextInput } from 'components/forms/TextInput'
+
+/**
+ *
+ */
+export const PanelEditor = ({ panel, onChange, currentLanguage, supportedLanguages }: {
+    panel: FormPanel, onChange: (newPanel: FormElement) => void,
+    currentLanguage: PortalEnvironmentLanguage, supportedLanguages: PortalEnvironmentLanguage[]
+}) => {
+  return (
+    <>
+      <TextInput className={'mb-2'} label={'Title'} value={panel.title}
+        onChange={title => onChange({ ...panel, title })}/>
+      <div>
+        {panel.elements.map((element, elementIndex) => {
+          const questionLabelId = useId()
+
+          return <div key={elementIndex}>
+            <CollapsibleSectionButton
```

Comment:
Maybe controversial, but since you're only confined to see one page at a time, I think I would personally prefer seeing these open by default - having some open and some collapsed when i opened the page confused me a little bit at first